appveyor.yml

```diff
@@ -33,11 +35,28 @@ before_build:
 
 build_script:
   - cmd: cmake --build build --config %CONFIGURATION% --target core -- /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" /verbosity:minimal /maxcpucount:2 /nologo
+  - cmd: |
+      7z a "hpx-master-redist-%MSVC_GENERATOR_SHORT%-%CONFIGURATION%.7z" "build\Debug\bin\*.dll" "build\Debug\bin\hpx\*.dll" "build\Debug\bin\*.pdb" "build\Debug\bin\hpx\*.pdb" >NUL
+      7z a "hpx-master-redist-%MSVC_GENERATOR_SHORT%-%CONFIGURATION%.7z" "build\Debug\lib\*.lib" "build\Debug\lib\hpx\*.lib" "build\src\hpx_init.dir\Debug\hpx_init.pdb" >NUL
+      7z a "hpx-master-redist-%MSVC_GENERATOR_SHORT%-%CONFIGURATION%.7z" "build\lib\cmake\HPX\*.cmake" "build\hpx\config\*.hpp" "build\hpx\runtime\parcelset\*.hpp" "hpx" "cmake\*.cmake" >NUL
   - cmd: cmake --build build --config %CONFIGURATION% --target examples -- /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" /verbosity:minimal /maxcpucount:2 /nologo
-#  - cmd: cmake --build build --config %CONFIGURATION% --target tests/regressions -- /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" /verbosity:minimal /maxcpucount:2 /nologo
 
 test_script:
   - cmd: |
       set PATH=%PATH%;C:\projects\vcpkg\installed\x64-windows\debug\bin
       python build\bin\hpxrun.py -l2 -t2 build\Debug\bin\hello_world.exe -- --hpx:bind=none
 
+artifacts:
+  - path: hpx-master-redist-$(MSVC_GENERATOR_SHORT)-$(configuration).7z
```

Comment:
What information does the configuration contain?

Reply:
@sithhell 'Debug', 'Release', etc. It's essentially the `CMAKE_BUILT_TYPE`.

Reply:
Ok. Does it make sense to at the commit hash?

Reply:
I intentionally didn't do that. All I need is the snapshot from the latest successful build. Similar to what we do in CircleCI.